Helpers/MediaLaunchObject.m

```diff
@@ -0,0 +1,46 @@
+//
+//  MediaLaunchObject.m
+//  ConnectSDK
+//
+//  Created by Ibrahim Adnan on 1/19/15.
+//  Copyright (c) 2015 LG Electronics. All rights reserved.
+//
+
+#import "MediaLaunchObject.h"
+
+@implementation MediaLaunchObject
+
+// return nil here so that we force users to have URL and mime-type
+- (instancetype) init
+{
+    return nil;
+}
+
+- (instancetype) initLaunchSession:(LaunchSession *)session withMediaControl:(id<MediaControl>)mediaControl
+{
+    self = [super init];
```

Comment:
I believe it's better to call the `init` method with more parameters here, passing these arguments to it. It will reduce code duplication.